.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "react"
+  ]
```

Comment:
A lot of the code you've written would fail in browsers other than recent versions of chrome. I'd recommend bringing in `babel-preset-es2015` as well. Plus, then you can be a cool.